### PR TITLE
Caro-Mo-Intro - blog post skeleton

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

**Comment by yabellini** [comment]
Hi, @caropradier and @drmowinckels here is the draft of your intro blog post.  I use the text and picture you already share with me.  You can review and made changes.  I think we can add some links to your projects and institutions if you want to.  

In case you can't review or suggest in this PR, please share the changes and/or link as comment here, and I will apply them on the blog post.

Thanks!!!

File: content/blog/2023-05-18-champions-program-teams/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+I applied to the Program in order to improve my skills and bring new tools to the social scientists’ communities in which I am actively involved and thus strengthen our capacities to transform reality. Furthermore, I intend to bring researchers from my country closer to the R community and hope to meet new professionals I can collaborate with in the future. Needless to say, I am particularly excited about the prospect of participating in a program aimed at making the R community more inclusive.
+
+</br>
+
+## Athanasia Monika Mowinckel - Mentor
+
+{{< figure src = "/img/team/mo-mentor.png" class = "pull-left" width = "300" alt = "Profile photo of Athanasia Monika Mowinckel" caption = "<center><strong>Athanasia Monika Mowinckel</strong></center>">}}
+
+
+My name Athanasia (Mo) Mowinckel, and I hold a PhD in cognitive neuroscience, and am based in Oslo Norway. I was born in Athens, Greece of a Cypriot mother and a Norwegian father, and moved to Norway with my family when I was 4. Here I now works as a staff scientist and Research Software Engineer at the Center for Lifespan Changes in Brain and Cognition, at the University of Oslo.
```
Comment (drmowinckels) [inline content/blog/2023-05-18-champions-program-teams/index.md:39]:
```suggestion
My name Athanasia (Mo) Mowinckel, and I hold a PhD in cognitive neuroscience, and am based in Oslo, Norway. I was born in Athens, Greece, of a Cypriot mother and a Norwegian father, and moved to Norway with my family when I was 4. Here, I now work as a staff scientist and Research Software Engineer at the [Center for Lifespan Changes in Brain and Cognition](https://www.oslobrains.no/) at the [University of Oslo](https://www.uio.no/english/). 
```

**Review by drmowinckels — CHANGES_REQUESTED** [review]
Just some minor details and links. Looks good. Thank you!

**Comment by yabellini** [comment]
@steffilazerte, this is also ready for your review.

File: content/blog/2023-05-18-champions-program-teams/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+title: "rOpenSci Champions Program Teams: Meet Carolina Pradier and Athanasia Monika Mowinckel"
+author:
+  - Carolina Pradier
+  - Athanasia Monika Mowinckel
```
Comment (steffilazerte) [inline content/blog/2023-05-18-champions-program-teams/index.md:5]: Both need author files

File: content/blog/2023-05-18-champions-program-teams/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+I am a mentor in the rOpenSci Champions program and am looking forward to learning about mentor roles and be inspired by new R users expanding their community roles. I'm hoping the Champions program will give me new insight into how a younger generation interacts with the community either as reviewers or as developers, and to learn with them about how we can open up and enabled more users to take part in our activities. I also look forward to connecting and interacting with other mentors and learning about how to be a good counsellor to others.
+
+
+_Carolina and Mo are already working on the eph package. we’re excited to see how it goes! Be sure to not miss the next series of introductions_
```
Comment (steffilazerte) [inline content/blog/2023-05-18-champions-program-teams/index.md:46]:
```suggestion
_Carolina and Mo are already working on the eph package. We’re excited to see how it goes! Be sure to not miss the next series of introductions_
```

File: content/blog/2023-05-18-champions-program-teams/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+
+My name Athanasia (Mo) Mowinckel, and I hold a PhD in cognitive neuroscience, and am based in Oslo, Norway. I was born in Athens, Greece, of a Cypriot mother and a Norwegian father, and moved to Norway with my family when I was 4. Here, I now work as a staff scientist and Research Software Engineer at the [Center for Lifespan Changes in Brain and Cognition](https://www.oslobrains.no/) at the [University of Oslo](https://www.uio.no/english/). 
+
+A lot of my work centers around creating data pipelines and work flows for my colleagues. In this work, I often choose to create R packages, which is one of the main programming languages in my field. I also create javascript applications and hugo websites on need depending on the project. I have been a reviewer for an rOpenSci package as well as for several [JOSS](https://joss.theoj.org/) submissions of R packages. I am currently part of the [R-Ladies Global leadership](https://www.rladies.org/about-us/global-team/), after 4 years in the R-Ladies global team as part of website development, and 5 years as chapter organiser in Oslo.
```
Comment (steffilazerte) [inline content/blog/2023-05-18-champions-program-teams/index.md:41]:
```suggestion
A lot of my work centers around creating data pipelines and work flows for my colleagues. In this work, I often choose to create R packages, which is one of the main programming languages in my field. I also create JavaScript applications and Hugo websites depending on the need of the project. I have been a reviewer for an rOpenSci package as well as for several [JOSS](https://joss.theoj.org/) submissions of R packages. I am currently part of the [R-Ladies Global leadership](https://www.rladies.org/about-us/global-team/), after 4 years in the R-Ladies Global team as part of website development, and 5 years as chapter organiser in Oslo.
```

File: content/blog/2023-05-18-champions-program-teams/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+I am a mentor in the rOpenSci Champions program and am looking forward to learning about mentor roles and be inspired by new R users expanding their community roles. I'm hoping the Champions program will give me new insight into how a younger generation interacts with the community either as reviewers or as developers, and to learn with them about how we can open up and enabled more users to take part in our activities. I also look forward to connecting and interacting with other mentors and learning about how to be a good counsellor to others.
```
Comment (steffilazerte) [inline content/blog/2023-05-18-champions-program-teams/index.md:43]:
```suggestion
I am a mentor in the rOpenSci Champions program and am looking forward to learning about mentor roles and to being inspired by new R users expanding their community roles. I'm hoping the Champions program will give me new insight into how another generation interacts with the community either as reviewers or as developers, and to learn with them about how we can open up and enable more users to take part in our activities. I also look forward to connecting and interacting with other mentors and learning about how to be a good counsellor to others.
```

**Review by steffilazerte — COMMENTED** [review]
Just some tweaks and author files needed :) 
* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok -> CHECK
* [X] alternative text of images is informative
* [ ] author metadata is provided with correct folder name -> **Need both**
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

File: content/blog/2023-05-18-champions-program-teams/index.md (hunk 2 above)
Comment (drmowinckels) [reply content/blog/2023-05-18-champions-program-teams/index.md:5]: check #557 for my author file

Comment (caropradier) [reply content/blog/2023-05-18-champions-program-teams/index.md:5]: find mine in #558

Comment (yabellini) [reply content/blog/2023-05-18-champions-program-teams/index.md:5]: Thanks!

**Pull request merged**